src/panels/lovelace/cards/hui-media-control-card.ts

```diff
@@ -125,28 +125,36 @@ export class HuiMediaControlCard extends LitElement implements LovelaceCard {
         <div
           class="background ${classMap({
             "no-image": !this._image,
-            "very-narrow": this._veryNarrow,
           })}"
         >
           <div
             class="color-block"
             style="background-color: ${this.backgroundColor}"
           ></div>
-          <div
-            class="color-gradient"
-            style="background-image: linear-gradient(to right, ${this
-              .backgroundColor}, transparent);"
-          ></div>
+          ${!this._image
+            ? ""
+            : html`
+                <div
+                  class="color-gradient"
+                  style="background-image: linear-gradient(to right, ${this
+                    .backgroundColor}, transparent);width: ${this
+                    ._cardWidth}px;"
```

Comment:
A div by default is a block element, which takes up 100% of the width, and so should be the same as `this._cardWidth` ?

Reply:
I name the variable incorrectly 😬

Reply:
It's the client height

Reply:
Can you then rename it to cardHeight